server.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var express = require('express');
+var bodyparser = require('body-parser');
+var Adjective = require('./lib/adjective.js');
+var Noun = require('./lib/noun.js');
+var Verb = require('./lib/verb.js');
+
+var getRandomWord = require('./lib/getRandomWord.js');
+var postRandomWord = require('./lib/postRandomWord.js');
+
+var app = express();
+var port = process.env.PORT || 3000;
+
```

Comment:
really good, clean server file. Does all the stuff it needs and none of what it doesn't.